sklearn/tree/_export.py

```diff
@@ -742,6 +742,26 @@ def recurse(self, node, tree, ax, max_x, max_y, depth=0):
             ax.annotate("\n  (...)  \n", xy_parent, xy, **kwargs)
 
 
+@validate_params(
+    {
+        "decision_tree": [DecisionTreeClassifier, DecisionTreeRegressor],
+        "out_file": "no_validation",
+        "max_depth": [Interval(Integral, 0, None, closed="left"), None],
+        "feature_names": [list, None],
```

Comment:
I am wondering if we were not supporting `"array-like"` without mentioning it here.

Reply:
We should run a small test.

Reply:
Okay, I will try. If `"array-like"` works, do we need to create test cases for it?

Reply:
Yes, it would be best to have a test to acknowledge the support.